lintPanel.js

```diff
@@ -65,13 +65,15 @@ define(function (require /*, exports, module*/) {
         $problemsPanelTable = $problemsPanel.find(".table-container");
         $problemsPanelTable.on("click", "tr", function () {
             var $target = $(this);
-            var line = $target.data("line") - 1; // Convert from friendly line to actual line number
-
+            var line = $target.data("line"); // Convert from friendly line to actual line number
```

Comment:
I think that removing the comment from this line seems like a good idea (or restoring the `- 1` after the query).